### ISSUE TITLE
Windows: unable to install gatsby-plugin-sharp, yet able to install sharp separately

### ISSUE THREAD

**Issue by Erbenos** [gatsbyjs/gatsby#22741]
## Description

I am unable to install `gatsby-plugin-sharp`. Seemed like `node-gyp` issue, but I am able to install `sharp` and similar native addons separately (eg `kerberos` https://pastebin.com/RxQiDw30, `bufferutil`).

### Steps to reproduce

Try installing `gatsby` with `gatsby-plugin-sharp`. My package.json contains this: `"gatsby":"2.0.91","gatsby-plugin-sharp": "2.2.14"` .

###  Expected result

Packages should install successfully.

### Actual result

Installation failure.

See: https://pastebin.com/SZZmMPhe
As said, I am able to install `sharp` separately: https://pastebin.com/1BBXat7T

Node v13.9.0

**Comment by Erbenos** [comment]
Jeez, was enough to update gatsby-plugin-sharp version, sorry for spam.

**Comment by Erbenos** [comment]
Well actually installation did somehow succeed, but running project via `npx gatsby develop` failed due to `sharp` anyway, even after updating `gatsby`, `gatsby-plugin-sharp` and `gatsby-transformer-sharp`.

**Comment by Js-Brecht** [comment]
Hey @Erbenos, can you try deleting `node_modules` and `$ENV:APPDATA\npm-cache\_libvips`, then rerunning the install?

**Comment by Erbenos** [comment]
I created new folders with `package.json` containing following deps:

```
"dependencies": {
    "gatsby":"2.0.91",
    "gatsby-plugin-sharp": "2.2.14"
  },
```

Result:
Downloads libvips, but still fails when building: https://pastebin.com/AWTB1V5v

**Comment by Js-Brecht** [comment]
Okay, when I use `node@13.9.0`, `gatsby@2.0.91`, and `gatsby-plugin-sharp@2.2.14`, I get the same error.

However, when I upgrade to `gatsby@2.20.12` and `gatsby-plugin-sharp@2.5.4`, installs works correctly, **and** I'm able to `develop`, `build` and also `serve` without any problems.

I see you updated at one point, but your last post shows old versions.  If you update to the current versions, can you verify that you still have the issue?

**Comment by moonmeister** [comment]
Sharp has a litany of compatibility issues...see any plugin docs that uses sharp and it'll talk you through them. Months ago there was a bunch of work done to get all the plugins  that depend on sharp updated and compatible...I believe there were some Gatsby core changes done as well.

TL;DR these are probably incompatible versions of Gatsby and sharp. Update everything and you'll be fine...if that's not an option upgrade or downgrade one or the other till everything works.

**Comment by LekoArts** [comment]
Thank you for opening this!

You should use the latest versions of both plugins mentioned as we recently bumped the minimum Node version to 10.13 and hence were able to also update sharp to a more recent version. Those version bumps included a lot of fixes for e.g. your problem or other more involved problems.

We're marking this issue as answered and closing it for now but please feel free to comment here if you would like to continue this discussion. We also recommend heading over to our [communities](https://www.gatsbyjs.org/contributing/community/#where-to-get-support) if you have questions that are not bug reports or feature requests. We hope we managed to help and thank you for using Gatsby!
